### ISSUE TITLE
Question about items not appearing in train set

### ISSUE THREAD

**Issue by miguelgfierro** [NVIDIA/DeepRecommender#15]
Hi @okuchaiev,

One question about the system. In the current [implementation](https://github.com/NVIDIA/DeepRecommender/blob/master/data_utils/movie_lense_data_converter.py#L85) if an item does not appear in the training set, it is removed. 

Is there other alternative to this? What would happen if we initialize the mappings with all train, validation and test set?

Thanks
